contracts/rust/src/bin/eval_domain.rs

```diff
@@ -0,0 +1,26 @@
+//! Binary to generate hardcoded parameters for EvaluationDomain supported in `PolynomicalEval.sol`
```

Comment:
```suggestion
//! Binary to generate hardcoded parameters for EvaluationDomain supported in `PolynomialEval.sol`
```

Reply:
done in [6171539](https://github.com/EspressoSystems/espresso-sequencer/pull/854/commits/61715390c68c49dcf05f32a2505fc782957d9628)